lib/clients/graphql/storefront_client.ts

```diff
@@ -41,13 +41,23 @@ export class StorefrontClient extends GraphqlClient {
 
   protected getApiHeaders(): HeaderParams {
     const sdkVariant = LIBRARY_NAME.toLowerCase().split(' ').join('-');
+    const usePrivateToken = this.storefrontClass().config.usePrivateToken;
+    const privateToken =
+      this.storefrontClass().config.privateAppStorefrontAccessToken;
 
+    if (!usePrivateToken && privateToken !== undefined) {
+      logger(this.storefrontClass().config).warning(
+        'Private storefront access token is ignored when usePrivateToken is false',
+      );
+    }
```

Comment:
We should probably log this when creating the app (maybe _in addition_ to here? Not sure if that won't be annoying) to inform the developer as early as possible.

Reply:
do you mean in the templates repo? or somewhere in this lib?

Reply:
I was thinking when the app calls `shopifyApi`, so it'll show up right as the server starts.